### ISSUE TITLE
Questions about Reverse Self-Contrastive Loss and LoRA Training

### ISSUE THREAD

**Issue by KOjuny** [tomguluson92/EraseAnything#5]
Hello, thank you for sharing this interesting work.
I have two questions that I could not fully understand from the paper:

1. Reverse Self-Contrastive (RSC) Loss

I don’t quite understand why the RSC loss should behave differently from the conventional self-contrastive loss.

In Equation (8), if we move the minus sign inside the log, it seems to become identical to Equation (9). Doesn’t this mean they are essentially the same loss function?

<img width="1392" height="146" alt="Image" src="https://github.com/user-attachments/assets/c8d756dc-7b2b-41b3-854c-385ea8d70275" />

<img width="1390" height="129" alt="Image" src="https://github.com/user-attachments/assets/f904a166-538d-4e11-b54b-a0d2da6635e2" />

2. LoRA Training Procedure

I would like to confirm if my understanding is correct:

- Is LoRA trained every time a new prompt 𝑐 is given?

- When applying the same concept to different prompt 𝑐, do we need to retrain LoRA for each 𝑐?

- If we want to remove multiple concepts for the same 𝑐, should we (a) train multiple LoRAs at once, or (b) train a separate LoRA for each concept and then apply all of them together at inference time?

I am not fully confident about how LoRA is used in this context, so I would appreciate a clear explanation.

Thank you very much for your help!
